### PR TITLE
docs: add docs for widgets

### PR DESCRIPTION
**Pull request by student-ice**
给darrowbutton添加文档注释#232

Log: add docs

Task: https://github.com/deepin-community/coding-quarter/issues/31

### CODE REVIEW

File: src/widgets/darrowbutton.cpp [hunk 1]
```diff
@@ -206,50 +205,43 @@ void DArrowButton::initButtonState()
...
   \value  ArrowStatePress
   Button got pressed state
-  按钮被按下状态
  */
 
 /*!
-  \brief Update arrow direction of the button
-  \brief 更新箭头按钮中箭头的方向
+@~english
+  \brief Update arrow direction of the button.
```
Comment (xzl01) [inline src/widgets/darrowbutton.cpp:244]: 这个位置要改为@brief

File: src/widgets/darrowbutton.cpp [hunk 2]
```diff
@@ -206,50 +205,43 @@ void DArrowButton::initButtonState()
 }
 
 /*!
+@~english
   \enum Dtk::Widget::DArrowButton::ArrowDirection
-  
-  \brief The ArrowDirection enum indicate the direction of the arrow icon in the arrown button
-  \brief DArrowButton::ArrowDirection 表示箭头图标的方向。
+
+  \brief The ArrowDirection enum indicate the direction of the arrow icon in the arrown button.
```
Comment (xzl01) [inline src/widgets/darrowbutton.cpp:212]: 需要进行修改

**Review by xzl01 — APPROVED** [review]

**Comment by student-ice** [comment]
merge

**Pull request merged**